src/bridge/MemberListSyncer.ts

```diff
@@ -250,8 +250,7 @@ export class MemberListSyncer {
         return this.syncableRoomsPromise;
     }
 
-    private joinMatrixUsersToChannels(rooms: RoomInfo[], injectJoinFn: InjectJoinFn) {
-        const d = promiseutil.defer();
+    private async joinMatrixUsersToChannels(rooms: RoomInfo[], injectJoinFn: InjectJoinFn) {
```

Comment:
Just want to ensure that the addition of `async` here is taken into account as I see no changes to any calling code.

Reply:
This function used to return a deferred (a Promise), and the caller was already awaiting it. We're now using await inside the function, but otherwise the return value remains the same.